app/Http/Controllers/Admin/DashboardController.php

```diff
@@ -41,6 +41,18 @@ public function index()
                 ->dimensions(config('custom.thousand'), config('custom.five_hundred'))
                 ->responsive(true);
 
+        for ($i = config('custom.three'); $i >= config('custom.zero'); $i--) {
```

Comment:
tại sao i = 3?

Reply:
dạ. em lấy kết quả trong 4 năm á a. vòng for của e chạy từ 3 đến 0